kubernetes/kubeclient.go

```diff
@@ -663,13 +671,54 @@ func InstallVamp(clientset *kubernetes.Clientset, ns string, config *models.Vamp
 	}
 	errDeployment := CreateOrUpdateDeployment(clientset, ns, vampDeployment)
 	if errDeployment != nil {
-		fmt.Printf("Warning: %v\n", errDeployment.Error())
+		fmt.Printf("Warning: error during deployment - %v\n", errDeployment.Error())
 		return nil, nil, nil, errDeployment
 	}
+
+	errHPA := CreateOrUpdateHPA(clientset, config)
+	if errHPA != nil {
+		fmt.Printf("Warning: error during hpa creation - %v\n", errHPA.Error())
+		return nil, nil, nil, errHPA
+	}
+
 	url := "https://" + ip + ":8888"
 	return &url, crt, key, nil
 }
 
+func CreateOrUpdateHPA(clientset *kubernetes.Clientset, config *models.VampConfig) error {
+	hpa := &autoscalingv1.HorizontalPodAutoscaler{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "vamp",
+			Labels: map[string]string{
+				"app":        "vamp",
+				"deployment": "vamp",
+			},
+		},
+		Spec: autoscalingv1.HorizontalPodAutoscalerSpec{
+			ScaleTargetRef: autoscalingv1.CrossVersionObjectReference{
+				Kind:       "Deployment",
+				Name:       "vamp",
+				APIVersion: "extensions/v1beta1",
+			},
+			MinReplicas:                    config.MinReplicas,
+			MaxReplicas:                    config.MaxReplicas,
+			TargetCPUUtilizationPercentage: config.TargetCPUUtilizationPercentage,
+		},
+	}
+	_, err := clientset.Autoscaling().HorizontalPodAutoscalers(InstallationNamespace).Create(hpa)
+	if err != nil {
+		fmt.Printf("Warning: got error while creating HPA - %v\nUpdating HPA...\n", err)
```

Comment:
Can you change this something like:
fmt.Printf("Warning: %v\n", err.Error())

We don't know if it will update HPA until it checks if it is a conflict error.

Reply:
Done